models/index.js

```diff
@@ -0,0 +1,4 @@
+import User from './User';
+import Product from './Product';
+
+module.exports = {User, Product};
```

Comment:
A mix of ES6 and ES5 module systems. Either use ES5 style
```
const User = require('./User');
module.exports = { User };
```

or ES6 style (according to the requirements)
```
import User from './User';
export { User };
```

or even such ES6 way
```
export { default as User } from './User';
```